conf/uppmax.config

```diff
@@ -1,8 +1,11 @@
-//Profile config names for nf-core/configs
+// Profile config names for nf-core/configs
 params {
   config_profile_description = 'Swedish UPPMAX cluster profile provided by nf-core/configs.'
   config_profile_contact = 'Phil Ewels (@ewels)'
   config_profile_url = 'https://www.uppmax.uu.se/'
+  project = null
+  clusterOptions = null
+  schema_ignore_params = "genomes,input_paths,cluster-options,clusterOptions,project"
```

Comment:
I don't think `genomes` and `input_paths` should be included here, but I like that

Reply:
The problem with `schema_ignore_params` is that it's just a string, so we have to overwrite it and can't just append to it as it stands. So these are here because they're there in the template and I didn't want to overwrite them.